ase/src/main/java/com/ase/ase/dao/CasesTimelineRepository.java

```diff
@@ -10,7 +11,7 @@
 public interface CasesTimelineRepository extends JpaRepository<CasesTimeline, Long> {
     @Query("SELECT new com.ase.ase.rest.response.CasesPerDate(t.time, t.newCases) FROM CasesTimeline t WHERE t.areaId = ?1")
     List<CasesPerDate> findAllBy(int areaId);
-
-    @Query(value = "SELECT b.sum_tested, c.sum_cases, c.sum_cases - c.sum_cured - c.sum_dead as current_sick, c.sum_dead FROM bed_and_test_timeline b, cases_timeline  c WHERE b.area_id = 10 and c.area_id = 10 and c.time = b.time order by b.time desc limit 1", nativeQuery = true)
-    Object[] getOverview();
+    //@Query(value = "SELECT b.sum_tested, c.sum_cases, c.sum_cases - c.sum_cured - c.sum_dead as current_sick, c.sum_dead FROM bed_and_test_timeline b, cases_timeline  c WHERE b.area_id = 10 and c.area_id = 10 and c.time = b.time order by b.time desc limit 1", nativeQuery = true)
+    @Query(value = "SELECT new com.ase.ase.rest.response.Overview(c.sum_cases - c.sum_cured - c.sum_dead as current_sick, c.sum_dead, c.sum_cases, c.sum_cured)  FROM bed_and_test_timeline b, cases_timeline  c WHERE b.area_id = 10 and c.area_id = 10 and c.time = b.time order by b.time desc limit 1", nativeQuery = true)
```

Comment:
This query will cause an server error.
It is a native query and therefore postgres runs it. Postgres does not know the overview class. So you have to write a native query which returns a json (with json_build_object()) or write a JPA Query which returns the 4 values and then make the json in the rest controler

Reply:
I have started to implement de db Methods, so you can use the method from the Branch DB-Methods for your REST-Call

Reply:
Very interesting that postgres doesn't know the overview class because i have imported like manuel did with the CasesPerDate. There is no problem but why then here?  Now I need to overlook how to transform from the json_build_object in the controller. I will for sure find out.

Reply:
As selene mentioned, you don't have to care about the db anymore. Just merge her branch DBmethods into yours and use her methods.